### DISCUSSION TITLE
新主页构建器前瞻

### DISCUSSION THREAD

**Discussion by Light-Beacon** [Light-Beacon/PCL2-NewsHomepage#112] [General]
已经能支持代码块了
![image](https://github.com/Light-Beacon/PCL2-NewsHomepage/assets/49828867/1b6a2ee0-249b-45de-a1fb-1c55c3b3ca0b)

**Comment by Light-Beacon** [comment]
顺带 100 星了 明天发个卡吼一声（bushi

**Comment by iPodtouch6557** [comment]
我觉得你把两个东西凑一起好长啊……

**Comment by SYBass** [comment]
老版的公开了？我找不到仓库了

**Reply by Light-Beacon** [reply]
https://github.com/Light-Beacon/PCL2-HomepageBuilder-Library
